ts/features/payments/checkout/screens/WalletPaymentPickMethodScreen.tsx

```diff
@@ -42,10 +45,14 @@ import {
   walletPaymentTransactionSelector
 } from "../store/selectors/transaction";
 import { FaultCodeCategoryEnum } from "../../../../../definitions/pagopa/ecommerce/GatewayFaultPaymentProblemJson";
+import { setAccessibilityFocus } from "../../../../utils/accessibility";
+import { WalletPaymentStepEnum } from "../types";
+import { PAYMENT_STEPS_TOTAL_PAGES } from "../utils";
 
 const WalletPaymentPickMethodScreen = () => {
   const dispatch = useIODispatch();
   const navigation = useIONavigation();
+  const titleRef = useRef<any>(null);
```

Comment:
Avoid the use of any
```suggestion
  const titleRef = useRef<View>(null);
```